operator/Dockerfile

```diff
@@ -24,4 +24,5 @@ WORKDIR /
 COPY --from=builder /workspace/manager .
 USER 65532:65532
 
+EXPOSE 8082
```

Comment:
Do we have to expose the port here? Isnt this simply optional?

Reply:
I looked it up in the official doc
```
The EXPOSE instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published.
```
Thus, yes it is just optional and I will remove it